utilities/transactions/transaction_lock_mgr.h

```diff
@@ -27,7 +27,7 @@ struct LockMap;
 struct LockMapStripe;
 
 class Slice;
-class TransactionDBImpl;
+class PessimisticTxnDB;
```

Comment:
Even though I like the longer name more :D I am fine with both names as long as we are consistent